### DISCUSSION TITLE
Deactivate elements outside of domain - how to set it up and is it possible to use regions?

### DISCUSSION THREAD

**Discussion by samfredriksson** [OpenDrift/opendrift#1092] [Q&A] [answered]
Hi,
I´m trying to deactivate elements outside the domain but have some difficulites to set it up. Presently as below

o = OceanDrift(loglevel=0)  # Set loglevel to 0 for debug information
o.deactivate_elements(o.elements.lat > 58.4815, reason='outside')

Someone that can give advice to set it up correctly?

I would also like to be able to specify regions as for instance different latitudes as a function of longitude (mulitple constraints) - is that possible?

//Sam

**Comment by knutfrode** [comment, marked as answer]
Hi,

The correct should be:
`o.set_config('drift:deactivate_north_of', 58.4815)`

But you can also have particles be deactivated outside of forcing model coverage by setting fallback value to None for a given parameter:
`o.set_config('environment:fallback:x_sea_water_velocity', None)`

To deactivate within a specific region is not directly possible, though some workaround should not be very difficult to construct.

**Reply by samfredriksson** [reply]
Thanks, got both options to work for me. It could be an intersting feature to use a mask or multiple lat/lon to deactivate but not a priority for now.
